cmd/gdsutil/main.go

```diff
@@ -1244,6 +1257,83 @@ func generateTokenKey(c *cli.Context) (err error) {
 	return nil
 }
 
+func migrateContacts(c *cli.Context) (err error) {
+	// Create a list of vasp contacts and model contacts
+	modelContacts := make(map[string]*models.Contact)
+	var vaspContacts []*pb.Contact
+
+	// iterate through all vasps in the database
+	iter := ldb.NewIterator(util.BytesPrefix([]byte(wire.NamespaceVASPs)), nil)
+	for iter.Next() {
+		vasp := new(pb.VASP)
+		if err = proto.Unmarshal(iter.Value(), vasp); err != nil {
+			iter.Release()
+			return cli.Exit(err, 1)
+		}
+		iter := models.NewContactIterator(vasp.Contacts, true, false)
+
+		// Iterate through all contacts on the vasp
+		for iter.Next() {
+			vaspContact, _ := iter.Value()
+			vaspContacts = append(vaspContacts, vaspContact)
+			modelContact, AlreadyExists := modelContacts[vaspContact.Email]
```

Comment:
We may need to make sure the email is normalized before putting it in the map, although if it's coming from the VASP hopefully it will already be.